Makefile

```diff
@@ -25,6 +25,12 @@ proto:
 	for dir in $$(git ls-files '*.proto' | xargs -n1 dirname | uniq); do \
 		protoc -I $$dir --go_out=plugins=grpc:$$dir $$dir/*.proto; \
 	done
+	cat grpclb/grpc_lb_v1/load_balancer.pb.go | \
```

Comment:
> grpclb.proto is now no longer a copy of the canonical version of the proto. This means updating it requires extra steps.

I didn't realize grpclb.proto was an upstream thing. That's a pickle. We can probably work around this by running the code generator twice; once with grpc and once without.

> load_balancer.pb.go is modified by sed, which is not ideal for several reasons (e.g. it's not based on the AST like gofmt would be).

Let's be practical here; doing these edits based on the AST is a lofty goal, but this sed-based automation is certainly better than the existing hand edits which are prone to rot and also are obviously not based on the AST.

> I can think of two low-effort options that would allow us to remove the sed commands until we can switch to using go generate in a later PR:

How does go generate solve anything?

> Leave the rest of the changes as-is, but skip the grpclb.proto file in make proto and delete load_balancer.proto.

Is that better? I think it's worse for reasons mentioned above.

> Remove the service from grpclb.proto and re-add grpclb/grpclb_server_generated.go.

Doesn't that contradict your other point about keeping a canonical copy of grpclb.proto?